Kimseho_assignment/app.js

```diff
@@ -0,0 +1,80 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+// prettier-ignore
+myDataSource.initialize() 
+.then(() => {
+  console.log("Data Source has been initialized!");
+});
+
+app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+//health check
+app.get("/ping", (req, res) => {
+  res.status(200).json({ message: "pong" });
+});
+
+//create a book
+app.post("/user_signup", async (req, res, next) => {
+  const { name, email, password, profile_image } = req.body;
+
+  //console.log(req)
+
+  await myDataSource.query(
+    `INSERT INTO users(
+      name, email, password, profile_image)
+     VALUES (?,?,?,?);
+    `,
+    [name, email, password, profile_image]
+  );
```

Comment:
SQL 쿼리문 포맷팅을 맞춰서 작성해주세요 !
```suggestion
  await myDataSource.query(
    `INSERT INTO users(
         name, 
         email, 
         password, 
         profile_image)
     VALUES (?,?,?,?);
    `,
    [name, email, password, profile_image]
  );
```